app/frontend/src/components/Divider.tsx

```diff
@@ -4,8 +4,8 @@ import makeStyles from "utils/makeStyles";
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    marginBottom: theme.spacing(1),
-    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+    marginTop: theme.spacing(3),
```

Comment:
I don't think it makes sense for the divider not to have the same margin either side. If there's unevenness somewhere, it should be fixed in the relevant spots. We want to try and not set margins on components as a rule (set the in layouts using `className` instead), because it affects things "outside" the component and can subtly break things. (Obviously here there has to be margin since it's more like setting the height)